src/components/EditTextModal/EditTextModal.js

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import { useSelector, useDispatch } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import { Swipeable } from 'react-swipeable';
+import { FocusTrap } from '@pdftron/webviewer-react-toolkit';
+
+import defaultTool from 'constants/defaultTool';
+import core from 'core';
+import actions from 'actions';
+import selectors from 'selectors';
+import AutoResizeTextarea from 'components/NoteTextarea/AutoResizeTextarea';
+
+import './EditTextModal.scss';
+
+const EditTextModal = () => {
```

Comment:
I think we should use normal functions rather than arrow functions when defining component. This make sure that in all dev tooling and debugging components will have names and they are not anonymous. React and babel might handle convert this, but using normal function makes sure that it is defined as named function rather than anonymous